templates/Angular2Spa/ClientApp/boot-client.ts

```diff
@@ -1,29 +1,22 @@
-import 'angular2-universal-polyfills/browser';
+import 'reflect-metadata';
+import 'zone.js';
```

Comment:
You could just have (I believe it's a bit smaller) `import 'zone.js/dist/zone-node';` here instead, just an idea!